lib/ddtrace/contrib/http_annotation_helper.rb

```diff
@@ -0,0 +1,10 @@
+module Datadog
+  module Contrib
+    # Contains methods helpful for tracing/annotating HTTP request libraries
+    module HttpAnnotationHelper
```

Comment:
Should we extract this?

Although it does reduce some repetition, it might be a bit premature given its this single line function. I might favor the repetition over having the additional constant in this case; the use of options specific to the integration (such as `service_name` and `split_by_domain` I think reinforces this right now, too. Eventually if the behavior grows in scope, an extraction like this might be appropriate.